src/integration-test/tests/citizen/claimantResponse/steps/claimant-reponse.ts

```diff
@@ -0,0 +1,135 @@
+import { ClaimantPaymentPlanPage } from 'integration-test/tests/citizen/claimantResponse/pages/claimant-payment-plan'
+import { ClaimantPayBySetDateAcceptedPage } from 'integration-test/tests/citizen/claimantResponse/pages/claimant-pay-by-set-date-accepted'
+import { ClaimantCounterOfferAcceptedPage } from 'integration-test/tests/citizen/claimantResponse/pages/claimant-counter-offer-accepted'
+
+const I: I = actor()
+const taskListPage: ClaimantTaskListPage = new ClaimantTaskListPage()
+const acceptPaymentMethodPage: ClaimantAcceptPaymentMethod = new ClaimantAcceptPaymentMethod()
+const chooseHowToProceedPage: ClaimantChooseHowToProceed = new ClaimantChooseHowToProceed()
+const checkAndSendPage: ClaimantCheckAndSendPage = new ClaimantCheckAndSendPage()
+const confirmationPage: ClaimantConfirmation = new ClaimantConfirmation()
+const signSettlementAgreementPage: ClaimantSignSettlementAgreement = new ClaimantSignSettlementAgreement()
+const ccjPaidAmountSummaryPage: ClaimantCcjPaidAmountSummaryPage = new ClaimantCcjPaidAmountSummaryPage()
+const ccjPaidAnyMoneyPage: ClaimantCcjPaidAnyMoneyPage = new ClaimantCcjPaidAnyMoneyPage()
+const paymentOptionPage: ClaimantPaymentOptionPage = new ClaimantPaymentOptionPage()
+const paymentDatePage: ClaimantPaymentDatePage = new ClaimantPaymentDatePage()
+const paymentPlanPage: ClaimantPaymentPlanPage = new ClaimantPaymentPlanPage()
+const counterOfferAcceptedPage: ClaimantCounterOfferAcceptedPage = new ClaimantCounterOfferAcceptedPage()
+const payBySetDateAcceptedPage: ClaimantPayBySetDateAcceptedPage = new ClaimantPayBySetDateAcceptedPage()
+const claimantRepaymentPlan: PaymentPlan = {
+  equalInstalment: 5.00,
+  firstPaymentDate: '2025-01-01',
+  frequency: 'everyMonth'
+}
+
+export class ClaimantResponseSteps {
+
+  acceptSettlementFromDashboardWhenRejectPaymentMethod (claimRef: string, paymentOption: PaymentOption): void {
+    this.viewClaimFromDashboard(claimRef, true)
+    this.acceptSettlement(false, paymentOption)
+  }
+
+  acceptSettlementFromDashboardWhenAcceptPaymentMethod (claimRef: string): void {
+    this.viewClaimFromDashboard(claimRef, true)
+    this.acceptSettlement(true, undefined)
+  }
+
+  acceptCcjFromDashboardWhenDefendantHasPaidNoneAndAcceptPaymentMethod (claimRef: string): void {
+    this.viewClaimFromDashboard(claimRef, true)
+    this.acceptCCJ(false)
+  }
+
+  acceptCcjFromDashboardWhenDefendantHasPaidSomeAndAcceptPaymentMethod (claimRef: string): void {
+    this.viewClaimFromDashboard(claimRef, true)
+    this.acceptCCJ(true)
+  }
+
+  viewClaimFromDashboard (claimRef: string, shouldRespondToOffer: boolean): void {
+    I.click('My account')
+    I.see('Your money claims account')
+    I.click(claimRef)
+    if (shouldRespondToOffer) {
+      I.click('View and respond to the offer')
+    }
+  }
+
+  acceptSettlement (shouldAcceptPaymentMethod: boolean, paymentOption: PaymentOption): void {
```

Comment:
maybe split this function into two, as function name suggests accepting settlement when one flow does accepSettlementAndPaymentMethod, rejectSettlementSuggestPaymentPlan